pwa/src/templates/wooItemDetailTemplate/WOOItemDetailTemplate.tsx

```diff
@@ -53,7 +53,7 @@ export const WOOItemDetailTemplate: React.FC<WOOItemDetailTemplateProps> = ({ wo
         {getItems.isSuccess && (

```

Comment:
I think it'd be more effective to check if the `.data` is available. Instead of checking the `.data?.prop` everytime in the rest of the file.